homeassistant/components/switch/rest.py

```diff
@@ -10,20 +10,25 @@
 import voluptuous as vol
 
 from homeassistant.components.switch import (SwitchDevice, PLATFORM_SCHEMA)
-from homeassistant.const import (CONF_NAME, CONF_RESOURCE)
+from homeassistant.const import (
+    CONF_NAME, CONF_RESOURCE, CONF_VALUE_TEMPLATE, CONF_TIMEOUT)
 import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers import template
 
 CONF_BODY_OFF = 'body_off'
 CONF_BODY_ON = 'body_on'
 DEFAULT_BODY_OFF = 'OFF'
 DEFAULT_BODY_ON = 'ON'
 DEFAULT_NAME = 'REST Switch'
+DEFAULT_TIMEOUT = 10
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_RESOURCE): cv.url,
     vol.Optional(CONF_BODY_OFF, default=DEFAULT_BODY_OFF): cv.string,
```

Comment:
These are now considered templates and should be validated as such. `cv.template`